### ISSUE TITLE
some question about training 2 phase

### ISSUE THREAD

**Issue by jucaizhai** [Abdullah-Abuolaim/multi-task-defocus-deblurring-dual-pixel-nimat#10]
hi,when I am according to the process ,configured the environment,while use the mdp_phase_1_wacv.hdf5 as the pretrained model,when training finished,I just get the PSNR=24.64 ,the parameter I changed **'ing_mini_b' from 8 to 6**,because of the memory limited.The other parameters I didn't change.The dataset I choose is DPDD dataset，350 pairs images as the training dataset.the GPU is 2 NVIDIA 1080Ti.I don't know why

**Comment by Abdullah-Abuolaim** [comment]
Hello there,

There should be something missing. Make sure all the parameters and training procedure is similar to what is described in the main paper. Another good practice, when reducing the mini-batch size, we need to increase the number of iterations or epochs. If training with more epochs doesn't help you get a PSNR around 25.35 dB, then there should be something missing.
I hope that works for you. Please let me know what you get.

Best,
Abdullah

**Comment by jucaizhai** [comment]
Hello,I increased the number of epoch from 350 to 480,and change the loss function's lambda from [1,1,0.2.0.2] to[1,0.8,0.5,0.5], while there has no performance increase.So I decided to use the model the git produce to evaluation.

Best wishes
